roles/sahara/defaults/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+sahara:
+  enabled: True
+  juno_url: http://tarballs.openstack.org/sahara/sahara-stable-juno.tar.gz
+  dashboard_url: http://tarballs.openstack.org/sahara-dashboard/sahara-dashboard-master.tar.gz
+  cafile: "{{ ssl.cafile|default('/etc/ssl/certs/ca-certificates.crt') }}"
+logs:
+    logging:
```

Comment:
Logging should be its own top level key, not a sub-element of logs.

Logs should have data about what log files sahara writes out. See other openstack service roles for examples.